releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,13 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.0.0-preview4 - September 13, 2023
+
+### New Features
+* Added support for periodically resolving Key Vault references to fetch latest version secret and update Kubernetes secret accordingly.
+* Added support for generating ConfigMap in more data types (json, yaml, properties).

```

Comment:
Added support for consuming the generated ConfgiMap as a mounted file besides as environment variables.

(It will be good to reference any relevant issues, if any)

Reply:
Done. Thanks for your suggestion.